.github/ISSUE_TEMPLATE/E-proponer-software-para-revision.md

```diff
@@ -22,8 +22,11 @@ Tipo de Envio: <!--submission-type-->pre-envio<!--end-submission-type-->
 
 ## Alcance 
 
-- Por favor, indica qué categoría(s) aplican a este paquete. Las puedes encontrar en nuestras [políticas de inclusión de paquetes (Inglés)](https://ropensci.github.io/dev_guide/policies.html#package-categories). Por favor, tilda todas las apropiadas. Si no estás seguro, te sugerimos que comiences un pre-envío.
+- Por favor, indica qué categoría(s) aplican a este paquete. Las puedes encontrar en nuestras [políticas de inclusión de paquetes (Inglés)](https://ropensci.github.io/dev_guide/policies.html#package-categories) y [categorías de paquetes de estadística (Inglés)](https://stats-devguide.ropensci.org/overview.html#overview-categories). Por favor, tilda todas las apropiadas. Si no estás seguro, te sugerimos que comiences un pre-envío.
```

Comment:
`^3:` Maybe wrap at 80 characters to make the diffs easier to see?

`^4:` Similar to `^2` I suggest (but not so sure):
"paquetes `[-de estadistica-]{+estadisticos+}`"